skt/executable.py

```diff
@@ -385,10 +387,15 @@ def setup_parser():
     """
     parser = argparse.ArgumentParser()
 
-    parser.add_argument("-d", "--workdir", type=str, help="Path to work dir")
+    parser.add_argument("-d", "--workdir", type=str,
+                        help="Path to work dir with kernel source git tree",
+                        default=os.environ.get("SKT_WORKDIR"))
```

Comment:
Could you please make adding the environment variable retrieval a separate commit as well?

Reply:
Done.